lib/Bio/KBase/ObjectAPI/KBaseFBA/FBAModel.pm

```diff
@@ -5,11 +5,11 @@
 # Development location: Mathematics and Computer Science Division, Argonne National Lab
 # Date of module creation: 2012-03-26T23:22:35
 ########################################################################
-use strict;
+package Bio::KBase::ObjectAPI::KBaseFBA::FBAModel;
```

Comment:
Moving package declaration to top of file. The rest of the changes to this file are whitespace.